docs/tutorials/01_introduction_cloud_runtime.ipynb

```diff
@@ -178,7 +178,7 @@
       "==================================================\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    print(job.backend)
</pre><p>Maybe add a short comment that explains why you print the backend here in addition to the result.</p><p> </p><p>At a first glance this looked like a left-over from local debugging to me.  Only when I continued reading in the notebook ("In the example above, we invoked a runtime program without specifying which backend it should run on. In this case the server automatically picks the one that is the least busy") I thought this might have been an intentional change.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Fixed.